modules/ROOT/pages/provisioning-nutanix.adoc

```diff
@@ -0,0 +1,70 @@
+= Provisioning Fedora CoreOS on Nutanix AHV
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Nutanix AHV. Fedora currently does not publish Fedora CoreOS images within Nutanix, so you must download a Nutanix image from Fedora and upload it to your Nutanix Prism Central subscription.
+
+== Prerequisites
+
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to a Nutanix Prism Central subscription. The examples below use the curl command to access Nutanix Prism Central APIs
+
+== Downloading a Nutanix image
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
```

Comment:
The download part isn't relevant.  Let's merge the first sentence and a half of this paragraph into the next section, and delete the rest of this section.